frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -220,9 +168,27 @@ function Pages(): JSX.Element {
                     />
                     <PageButton icon={<IconCohort />} identifier={Scene.Cohorts} to={urls.cohorts()} />
                     <PageButton icon={<IconComment />} identifier={Scene.Annotations} to={urls.annotations()} />
-                    <LemonDivider />
-                    {canViewPlugins(currentOrganization) && (
-                        <PageButton icon={<IconExtension />} identifier={Scene.Plugins} to={urls.plugins()} />
+                    {featureFlags[FEATURE_FLAGS.FRONTEND_APPS] ? (
+                        <>
+                            <div className="SideBar__heading">Apps</div>
+                            {canViewPlugins(currentOrganization) && (
+                                <PageButton
+                                    title="Browse Apps"
+                                    icon={<IconApps />}
+                                    identifier={Scene.Plugins}
+                                    to={urls.plugins()}
+                                />
+                            )}
+                            <SideBarApps />
```

Comment:
non-blocker: just leaving this here based on our sync discussion - when (under our current setup) do I see apps but can't use the apps browser/config page?

Reply:
👍 , bug fixed

The apps themselves should remain accessible for all users, even if they can't see the "browse apps" page for whatever reason (all I know is the API can say you have no access to the page 🤷). In this restricted case, an admin sets up the apps, and you as a user can then use them.

We'll need to go granular than that eventually, but not in this feature flagged v0.1.